.github/workflows/bench_reports.yml

```diff
@@ -1,6 +1,9 @@
 name: Benchmark Reports
 
 on:
+
+  pull_request:
+    branches: [main]
```

Comment:
This causes a bottleneck on our benchmark machine, let's not do this

Reply:
you can run this manually, we don't need to make temporary changes like this i think